server/src/models/User.ts

```diff
@@ -0,0 +1,37 @@
+import { ObjectId, Schema, Model, model } from "mongoose";
+
+export interface UserAttributes {
+  saved_restaurants: ObjectId;
+}
+
+export interface UserDocument extends UserAttributes {
+  _id: ObjectId;
+  auth0_id: string;
+  created_at: Date;
+}
+
+export const UserSchema: Schema<UserDocument> = new Schema<UserDocument>({
+  _id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    auto: true,
+  },
+  auth0_id: {
+    type: String,
+    required: true,
+  },
+  saved_restaurants: {
+    type: Schema.Types.ObjectId,
```

Comment:
this should be an array. wrap it in brackets like how each menu has an array of food ObjectIds

from models/Menu.ts:
```ts
export const MenuSchema: Schema<MenuDocument> = new Schema<MenuDocument>({
  ...
  foods: {
    type: [Schema.Types.ObjectId],
    required: true,
    ref: "Food",
  },
  ...
  ```